recipes/default.rb

```diff
@@ -18,4 +18,4 @@
 # limitations under the License.
 # 
 
-include_recipe "plexapp::server_#{node['platform']}"
+include_recipe "plexapp::server"
```

Comment:
Having this be an attribute could be okay, and the new server recipe would be better suited as a "direct_download" or similar.